src/features/processPicker.ts

```diff
@@ -246,4 +333,16 @@ function execChildProcess(process: string, workingDirectory: string): Promise<st
             resolve(stdout);
         });
     });
+}
+
+function execChildProcessWithTimeout(process: string, workingDirectory: string): Promise<string> {
+    return new Promise<string>((resolve, reject) => {
+        child_process.exec(process, { timeout: 3000, cwd: workingDirectory, maxBuffer: 500 * 1024 }, (error: Error, stdout: string, stderr: string) => {
```

Comment:
Will this command be async? Is there a way to cancel it? Its always nice to not have timeouts at all, as it is always hard to pick a good one. If you need one, I would probably suggest picking something pretty long since this involves connecting to a remote machine, which could be fairly slow. Maybe 30 seconds?